src/app/src/test/kotlin/org/imperial/mrc/hint/integration/IntegrationTests.kt

```diff
@@ -34,7 +34,10 @@ abstract class SecureIntegrationTests : CleanDatabaseTests()
     @BeforeEach
     fun beforeEach(info: TestInfo)
     {
-        userRepo.addUser("guest", "guest")
+        val userExists = userRepo.getUser("guest")
+        if (userExists == null) {
+            userRepo.addUser("guest", "guest")
+        }
```

Comment:
Added this check as it was causing the tests to fail if I had previously used the running app or run a test before. Let me know if this is not ok?